mne_bids/mne_bids.py

```diff
@@ -550,18 +552,25 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     bids_basename : str
         The base filename of the BIDS compatible files. Typically, this can be
         generated using make_bids_basename.
-        Example: sub-01_ses-01_task-testing_acq-01_run-01
-        This will write the following files in the correct subfolder
-        of output_path:
+        Example: `sub-01_ses-01_task-testing_acq-01_run-01`.
+        This will write the following files in the correct subfolder of the
+        output_path::
+
             sub-01_ses-01_task-testing_acq-01_run-01_meg.fif
             sub-01_ses-01_task-testing_acq-01_run-01_meg.json
             sub-01_ses-01_task-testing_acq-01_run-01_channels.tsv
             sub-01_ses-01_task-testing_acq-01_run-01_coordsystem.json
+
         and the following one if events_data is not None
+        ::
```

Comment:
could go to previous line?

Reply:
Thanks for this fix btw :) It was long due

Reply:
oh yeah, I was just leaving the formatting as it was before, but putting this on the previous line will give it a colon after the `None`. Probably better to have it that way.